src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/FhirDataStore.cs

```diff
@@ -211,6 +214,49 @@ public async Task HardDeleteAsync(ResourceKey key, CancellationToken cancellatio
             }
         }
 
+        public async Task<HttpStatusCode> UpsertExportJobAsync(ExportJobRecord jobRecord, CancellationToken cancellationToken = default)
```

Comment:
nit: Remove the default value from cancellationToken. We should do a cleanup when we have chance.